helm/grafana/values.yaml

```diff
@@ -25,35 +41,38 @@ grafana:
     create: true
     name:
     nameTest:
-  ## Service account annotations. Can be templated.
-  #  annotations:
-  #    eks.amazonaws.com/role-arn: arn:aws:iam::123456789000:role/iam-role-name-here
-    autoMount: true
+    ## ServiceAccount labels.
+    labels: {}
+    ## Service account annotations. Can be templated.
+    #  annotations:
+    #    eks.amazonaws.com/role-arn: arn:aws:iam::123456789000:role/iam-role-name-here
+
+    ## autoMount is deprecated in favor of automountServiceAccountToken
+    # autoMount: false
+    automountServiceAccountToken: false
 
   replicas: 1
 
   ## Create a headless service for the deployment
   headlessService: false
 
+  ## Should the service account be auto mounted on the pod
+  automountServiceAccountToken: true
```

Comment:
Why is there 2 `automountServiceAccountToken` ? What's the difference between `grafana.serviceAccount.automountServiceAccountToken` and `grafana.automountServiceAccountToken` ?

Reply:
You know I just copied the file from upstream right? :D